websites/C/Cineby/presence.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable camelcase */
+
+import { CinebyApi, MovieDetails, TvDetails } from "./api";
+
+const presence = new Presence({
+	clientId: "1325115346696273993",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/zAKeytL.png",
+			details: "Browsing",
+			type: ActivityType.Watching,
+		},
+		{ pathname } = document.location,
+		[showBrowsing, usePresenceName, showCover] = await Promise.all([
+			presence.getSetting<boolean>("showBrowsing"),
+			presence.getSetting<boolean>("usePresenceName"),
+			presence.getSetting<boolean>("showCover"),
+		]);
+
+	switch (pathname.split("/")[1]) {
+		case "movie": {
+			const { title, poster_path, release_date, runtime } =
+				await CinebyApi.getCurrent<MovieDetails>(pathname);
+
+			if (title) {
+				if (usePresenceName) presenceData.name = title;
+				presenceData.details = title;
+				presenceData.state = `${release_date
+					.split("-")
+					.shift()} • ${runtime} minutes`;
+
+				if (showCover)
+					presenceData.largeImageKey = `https://image.tmdb.org/t/p/original${poster_path}`;
+			}
+			break;
+		}
+		case "tv": {
+			const {
+				name: title,
+				season_poster,
+				episode_title,
+				season_number,
+				episode_number,
```

Comment:
You might be able to avoid needing to ignore camel case by renaming the properties.
```suggestion
				season_poster: seasonPoster,
				episode_title: episodeTitle,
				season_number: seasonNumber,
				episode_number: episodeNumber,
```